Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: clean-pyc clean-build clean-test test clean
```

Comment:
I'm waiting on @mattsfuller to give me the green light to put Apache licenses on all the files.